astroquery/esa/hubble/core.py

```diff
@@ -696,5 +773,11 @@ def _getCoordInput(self, value):
         else:
             return value
 
+    def _get_decoded_string(self, string):
```

Comment:
Somehow I cannot reply to the previous thread on this. Thanks for making this "private". In the longer term I think it's useful to move these to a shared utils for esa. But that's point beyond the PR.